uint8 id
uint8 cmd_mode


